nengo/builder/connection.py

```diff
@@ -54,6 +65,18 @@ def build_linear_system(model, conn, rng):
     return eval_points, activities, targets
 
 
+def multiply(x, y):
```

Comment:
Do you want a docstring because every function should have a docstring, or because there's something about this function that you think is unclear? Other helper functions in this file do not have docstrings, but I'm happy to add one if things need clarifying.